<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExternalToolParameterDialog</class>
 <widget class="QDialog" name="ExternalToolParameterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_excelPath">
      <property name="text">
       <string>Veuillez renseigner le chemin vers Excel :</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_excelPath"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_parcourirExcel">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_cppCheckPath">
      <property name="text">
       <string>Veuillez renseigner le chemin vers CPPCheck :</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_cppCheckPath"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_parcourirCPPCheck">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_checkStylePath">
      <property name="text">
       <string>Veuillez renseigner le chemin vers CheckStyle :</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEdit_checkStylePath"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_parcourirCheckStyle">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_valider">
        <property name="text">
         <string>Enregistrer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_annuler">
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
